test/operators/expression/arithmetic.test.ts

```diff
@@ -96,22 +98,22 @@ support.runTest("operators/expression/arithmetic", {
     [[-45.39, -1], -50],
     // rounded to the whole integer
     [[19.25, 0], 19],
-    [[28.73, 0], 28],
+    [[28.73, 0], 29],
     [[34.32, 0], 34],
     [[-45.39, 0], -45],
     [[1.6016, 3], 1.602],
-    [[1.6015, 3], 1.601],
+    [[1.6015, 3], 1.601]
```

Comment:
Please include these tests for future reference.

```
    // https://github.com/kofrasa/mingo/issues/347
    [[10.4, 0], 10],
    [[10.5, 0], 10],
    [[10.6, 0], 11],
    [[10.7, 0], 11],
    [[11.4, 0], 11],
    [[11.5, 0], 12],
    [[11.9, 0], 12]
```

Reply:
Added tests cases. 10.5 and 11.5 are already in the above test so  I did not duplicate them.